engine/polyglot-api/src/test/scala/org/enso/polyglot/ModuleManagementTest.scala

```diff
@@ -35,13 +36,23 @@ class ModuleManagementTest extends AnyFlatSpec with Matchers {
     def writeMain(contents: String): Unit = {
       Files.write(pkg.mainFile.toPath, contents.getBytes): Unit
     }
-  }
 
-  val subject = "Modules Polyglot API"
+    def teardown(): Unit = {
+      FileUtils.deleteDirectory(tmpDir)
```

Comment:
Same. If you're not able to mix the `WithTemporaryDirectory` here, you can extract `robustDeleteDirectory` into an object (make it static) and use it here directly